.github/workflows/cpp.yml

```diff
@@ -98,8 +98,8 @@ jobs:
             image: ubuntu-cpp
             llvm: 10
             runs-on: ubuntu-24.04-arm
-            title: ARM64 Ubuntu 20.04 C++
-            ubuntu: 20.04
+            title: ARM64 Ubuntu 22.04 C++
```

Comment:
The CI failure is related. We can't seem to be able to use llvm and clang 10 for Ubuntu 22.04. Should we bump to 14?

Reply:
Ah, you're right. I'll update them too.